portfolio.css

```diff
@@ -0,0 +1,279 @@
+  }
+
+  .modal-2 {
+    list-style: none;
+    background-color: #e5e5e5;
+    color: #6070ff;
+    border: 1px solid #e5e5e5;
+    border-radius: 8px;
+    padding: 8px 12px;
+    font-size: 15px;
+  }
+
+  .modal-3 {
+    list-style: none;
+    width: 100%;
+    background-color: #e5e5e5;
+    color: #6070ff;
+    border: 1px solid #e5e5e5;
+    border-radius: 8px;
+    padding: 8px 12px;
+    font-size: 15px;
+    text-align: center;
+  }
+}
+
+@media (min-width: 768px) {
+  .images-fit {
+    width: 100%;
+    height: 100%;
+  }
+
+  .pop-close {
+    display: block;
+    position: absolute;
+    top: 20px;
+    right: 20px;
+    color: var(--color-primary);
+  }
+
+  .tonic-1 {
+    height: 56px;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 700;
+    font-size: 40px;
+    color: #091e42;
+    padding-top: 24px;
+  }
+
+  .card-portfolio-1 {
+    background-color: #fff;
+    border-radius: 16px;
+    box-shadow: 0 0 5px rgba(0, 0, 0, 0.3);
+    width: 50%;
+    padding: 24px;
+    margin: 24px auto;
+    display: none;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    overflow: hidden;
+  }
+
+  .card-portfolio-1 span {
+    float: right;
+    padding-top: 24px;
+    padding-left: 24px;
+  }
+
+  .forM-1 {
+    display: none;
+  }
+
+  .forD-1 {
+    align-items: center;
+    margin-right: 20px;
+    width: 100%;
+  }
+
+  .heading-canopy-1 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  * {
+    box-sizing: border-box;
+  }
+
+  .row-1 {
+    padding-top: 59px;
+  }
+
+  .column-left-1 {
+    float: left;
+    width: 60%;
+    padding: 10px;
+  }
+
+  .column-left-1 p {
+    position: static;
+    width: 100%;
+    height: 100%;
+    left: 0;
+    top: 0;
+    font-family: 'Poppins', sans-serif !important;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    color: #344563;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin: 12px 0;
+  }
+
+  .column-right-1 {
+    float: left;
+    width: 40%;
+    padding: 10px;
+    height: 300px;
+  }
+
+  .row-1::after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+
+  hr {
+    position: fixed;
+    width: 298px;
+    border: 1px solid #ebecf0;
+    top: 24px;
+  }
+}
+
+.button-live-1 {
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+  border-radius: 8px;
+  color: #6070ff;
+  width: 142px;
+  display: initial;
+}
+
+.modal-1 {
+  color: #6070ff;
+  display: flex;
+  gap: 4px;
+  margin-top: 20px;
+  font-size: 12px;
+}
+
+.modal-2 {
+  list-style: none;
+  background-color: #e5e5e5;
+  color: #6070ff;
+  border: 1px solid #e5e5e5;
+  font-size: 15px;
+  height: 24px;
+  width: 93px;
+  left: 135px;
+  top: 0;
+  border-radius: 8px;
+  padding: 4px 12px 4px 12px;
+}
+
+.modal-3 {
+  list-style: none;
+  background-color: #e5e5e5;
```

Comment:
- [ ] Kindly make sure that the popup window is showing all the details of the projects. as it is, part of the project details are cut off, and the modal is not scrollable. kindly fix it.
![Capture](https://user-images.githubusercontent.com/85095702/162289201-ceb80943-6797-4935-a73e-e328a722023a.PNG)

Reply:
![Screenshot 2022-04-07 at 23 33 47](https://user-images.githubusercontent.com/9010615/162291265-8d4d91aa-356f-473a-ab6c-46c015261656.png)
This appears differently depending on the browser. Its browser compatibility

Reply:
@phelian23 I am unable to control it